src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -136,68 +137,74 @@ const Task = ({
     const TaskTypeIcon = task_type === TASK_TYPE_APPROVAL ? IconTaskApproval : IconTaskGeneral;
 
     return (
-        <div
-            className={classNames('bcs-task', {
-                'bcs-is-pending': isPending || error,
-            })}
-            data-testid="task-card"
-        >
-            <Comment
-                avatarRenderer={avatar => (
-                    <div className="bcs-task-avatar">
-                        {avatar}
-                        <TaskTypeIcon width={20} height={20} className="bcs-task-avatar-badge" />
-                    </div>
-                )}
-                created_at={created_at}
-                created_by={created_by.target}
-                currentUser={currentUser}
-                error={error}
-                id={id}
-                inlineDeleteMessage={messages.taskDeletePrompt}
-                isPending={isPending}
-                onDelete={onDelete}
-                onEdit={onEdit}
-                permissions={taskPermissions}
-                tagged_message={name}
-                translatedTaggedMessage={translatedTaggedMessage}
-                translations={translations}
-                getAvatarUrl={getAvatarUrl}
-                getUserProfileUrl={getUserProfileUrl}
-                mentionSelectorContacts={mentionSelectorContacts}
-                getMentionWithQuery={getMentionWithQuery}
-                userHeadlineRenderer={userLinkInstance => {
-                    return (
-                        <FormattedMessage
-                            {...getMessageForTask(!!currentUserAssignment, task_type)}
-                            values={{ user: userLinkInstance }}
+        <div className="bcs-task-container">
+            {error ? <CommentInlineError {...error} /> : null}
```

Comment:
instead of returning null, 
could you just use 
`` error && <CommentInlineError {...error} />```?